.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
 
-*                                    @MetaMask/devs
```

Comment:
I noticed that `MetaMask/devs` has been removed from the CODEOWNERS file. Could you provide some insight into the reason behind this change? I'm interested in understanding how this will impact the approval process for changes.

Reply:
We're trying to move away from broad ownership of repositories and instead assign them to specific teams.

If the repository is configured correctly in GitHub, this codeowner file should mean that all PRs need an approval from someone in our team before merging.